developments_build/03_export.sh

```diff
@@ -34,10 +34,9 @@ mkdir -p output
     CSV_export CORR_hny_matches &
     CSV_export applied_corrections &
     CSV_export not_applied_corrections &
-    CSV_export housing_input_research 
+    CSV_export manual_corrections 
```

Comment:
housing_input_research will be manual_corrections now?

Reply:
We are appending information about what got applied for HED, as requested in issue #359  https://github.com/NYCPlanning/db-developments/blob/8f3c892251e5758ff7e07ae74e4eb888ae8b438d/developments_build/sql/final.sql#L231-L248